Mathlib/Algebra/Order/AbsoluteValue/Basic.lean

```diff
@@ -373,6 +381,33 @@ lemma IsNontrivial.exists_abv_lt_one (h : v.IsNontrivial) : ∃ x ≠ 0, v x < 1
   rw [map_inv₀]
   exact (inv_lt_one₀ <| v.pos hy₀).mpr hy
 
+theorem isNontrivial_iff_exists_abv_one_lt :
+    v.IsNontrivial ↔ ∃ x, 1 < v x := by
+  refine ⟨fun h => h.exists_abv_gt_one, fun ⟨x, hx⟩ => ⟨x⁻¹, ?_, ?_⟩⟩
+  · simpa only [ne_eq, inv_eq_zero] using fun h ↦ v.map_zero ▸ not_le.2 (h ▸ hx) <| zero_le_one' _
+  · simpa only [map_inv₀, ne_eq, inv_eq_one] using ne_of_gt hx
+
+theorem inv_lt_one_iff {x : R} : v x⁻¹ < 1 ↔ x = 0 ∨ 1 < v x := by
+  simp [map_inv₀, inv_lt_one_iff₀, map_eq_zero]
+
+variable {w : AbsoluteValue R S}
+
+theorem one_lt_of_lt_one_imp (h : ∀ x, v x < 1 → w x < 1) {x : R} (hv : 1 < v x) : 1 < w x :=
```

Comment:
Isn't there a name mathematically for `∀ x, v x < 1 → w x < 1`? It corresponds to the valuations having the same integers, right? Maybe you should give it a name